tests/wicked/startandstop/sut/t09_openvpn_tunnel_ifdown.pm

```diff
@@ -15,23 +15,18 @@
 use base 'wickedbase';
 use strict;
 use testapi;
+use network_utils 'ifc_exists';
 
 sub run {
     my ($self) = @_;
     my $config = '/etc/sysconfig/network/ifcfg-tun1';
     $self->get_from_data('wicked/ifcfg/tun1_sut', $config);
     $self->setup_openvpn_client('tun1');
     $self->setup_tuntap($config, 'tun1', 0);
```

Comment:
remove parameter '0' from this call as it's not needed any more

Reply:
done